code/go/0chain.net/smartcontract/stakepool/unlock.go

```diff
@@ -23,6 +23,24 @@ func (sp *StakePool) UnlockPool(
 	if err != nil {
 		return 0, fmt.Errorf("can't get user pools list: %v", err)
 	}
+	return sp.UnlockPoolI(
+		clientID,
+		providerType,
+		providerId,
+		poolId,
+		usp,
+		balances,
+	)
+}
+
+func (sp *StakePool) UnlockPoolI(
```

Comment:
Rename this to UnlockPool and rename the above UnlockPool to something different perhaps, UnlockPoolI is meaningless.